lib/runner.js

```diff
@@ -228,7 +228,9 @@ module.exports = {
     log.info("envs", "Starting with queue size: " + chalk.magenta(queue || "unlimited"));
     map(function (taskEnv, cb) {
       // Add each specific set of environment variables.
-      var taskShOpts = _.merge({}, shOpts, { env: taskEnv });
+      // Clone `shOpts` to turn `env` into a plain object: in Node 4+
+      // `process.env` is a special object which changes merge behavior.
+      var taskShOpts = _.merge(_.cloneDeep(shOpts), { env: taskEnv });
```

Comment:
I wanted to use `_.toPlainObject(shOpts)` for clarify, but it doesn't work because `shOpts` itself *is* a plain object; the bogus `process.env` still gets through (plain-ness isn't recursive apparently).